code/modules/power/groundmap_geothermal.dm

```diff
@@ -192,6 +192,9 @@
 	start_processing()
 	return TRUE
 
+/obj/machinery/power/geothermal/attack_ai(mob/living/silicon/ai/user)
+	attack_hand(user)
```

Comment:
Do not do this, make them use a shared proc or such instead

Reply:
Done